src/constants/index.js

```diff
@@ -10,4 +10,10 @@ const APPS = {
 	LOAD_METRICS_FAIL: 'APPS_LOAD_METRICS_FAIL',
 };
 
-export { USER, APPS };
+const CREATE_APP = {
+	LOAD: 'CREATING_APP',
+	CREATE_APP_SUCCESS: 'CREATE_APP_SUCCESS',
```

Comment:
The term `CREATE_APP` not required in the keys here. Please check `USER` constant for reference.